packages/frontend-2/lib/projects/composables/team.ts

```diff
@@ -27,22 +27,17 @@ export function useTeamManagePermissionsInternals(params: {
   }
 }
 
-export function useTeamDialogInternals(params: {
-  props: {
-    project: Ref<ProjectPageTeamDialogFragment>
-  }
-}) {
-  const {
-    props: { project }
-  } = params
+export function useTeamInternals(
+  projectData: ComputedRef<ProjectPageSettingsCollaboratorsQuery | undefined>
```

Comment:
Ideally this should only ask for a fragment of what it needs. That way it can be re-used in more pages/components without having to request the entire collaborators query.

I already saw duplicate definitions of `isOwner` checks in various components